components/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -1,13 +1,13 @@
 // Copyright (c) Gridiron Survivor.
 // Licensed under the MIT License.
 
-import React from 'react';
+import { JSX } from 'react';
```

Comment:
Revert this. Deconstructing JSX directly is correct as we don't need all of React's dependencies.

Reply:
Updated in latest commit.

Can you elaborate on this? I thought using `JSX.Element` would be _more_ direct than using `React.JSX.Element`.

Reply:
> Updated in latest commit.
> 
> Can you elaborate on this? I thought using `JSX.Element` would be _more_ direct than using `React.JSX.Element`.

@shashilo pinging you for a reminder, please advise.